test/integration/integration-test.js

```diff
@@ -80,8 +86,16 @@ function teardown(done) {
 describe.skip('integration tests', function() {
   const instructions = {maxLedgerVersionOffset: 100};
   this.timeout(TIMEOUT);
+  const transactions = [];
 
+  // beforeEach(setup);
   beforeEach(setup);
+  beforeEach(function() {
+    if (!this.startLedgerVersion) {
```

Comment:
This code intends to initialize `this.startLedgerVersion` only one, very first time.
And I just realized that `this` object that gets passed to each `it` test is the same one, so you right, I can move this to `setup`.